coderd/database/migrations/000283_provisioner_daemon_status.up.sql

```diff
@@ -0,0 +1,3 @@
+CREATE TYPE provisioner_daemon_status AS ENUM ('offline', 'idle', 'busy');
```

Comment:
Do we need to create extra indices for `provisioner_daemon`s? Thinking about our scaletesting plans

Reply:
I find that it becomes apparent when performing scaletests when an index is needed. We can probably add them later as required.